dotnet/src/SemanticKernel.UnitTests/PromptTemplate/PromptTemplateConfigTests.cs

```diff
@@ -187,4 +188,81 @@ public void DeserializingExpectOutputVariable()
         Assert.NotNull(promptTemplateConfig.OutputVariable);
         Assert.Equal("output variable description", promptTemplateConfig.OutputVariable.Description);
     }
+
+    [Fact]
+    public void ItShouldDeserializeConfigWithDefaultValueOfStringType()
+    {
+        // Arrange
+        static string CreateJson(object defaultValue)
+        {
+            var obj = new
+            {
+                description = "function description",
+                input_variables = new[]
+                {
+                    new
+                    {
+                        name = "name",
+                        description = "description",
+                        @default = defaultValue,
+                        isRequired = true
+                    }
+                }
+            };
+
+            return JsonSerializer.Serialize(obj);
+        }
+
+        // string
+        var json = CreateJson((string)"123");
+        var config = PromptTemplateConfig.FromJson(json);
+
+        Assert.NotNull(config?.InputVariables);
+        Assert.Equal("123", config.InputVariables[0].Default?.ToString());
+    }
+
+    [Fact]
+    public void ItShouldThrowExceptionWhenDeserializingConfigWithDefaultValueOtherThanString()
```

Comment:
nit: Would be good to have a comment here to state this is a temporary limitation.

Reply:
Done